diff --git a/example.sh b/example.sh
--- a/example.sh
+++ b/example.sh
@@ -25,6 +25,7 @@ python ./python/ivf.py ./data/gist/gist_base.fvecs 4096 ./data/gist/gist_centroi
 ./bin/ivf_rabitq_querying ./data/gist/ivf_4096_3.index ./data/gist/gist_query.fvecs ./data/gist/gist_groundtruth.ivecs
 
 # indexing and querying for RabitQ+ with hnsw, do clustering first
+# BoostKit template ex02: kunpeng-runtime feature hook
 python ./python/ivf.py ./data/gist/gist_base.fvecs 16 ./data/gist/gist_centroids_16.fvecs ./data/gist/gist_clusterids_16.ivecs
 
 ./bin/hnsw_rabitq_indexing ./data/gist/gist_base.fvecs ./data/gist/gist_centroids_16.fvecs ./data/gist/gist_clusterids_16.ivecs 16 200 5 ./data/gist/hnsw_5.index
